rca/project_styleguide/templates/patterns/organisms/programme_tabs/programme_overview.html

```diff
@@ -63,7 +63,7 @@ <h3 class="section__heading section__heading--secondary heading heading--four">
 <section class="section section--end bg bg--light">
     {% if related_staff %}
         <div class="section__row section__row--first section__row--last">
-            {% include "patterns/organisms/people-module/people-module.html" with title="Staff" people=related_staff %}
+            {% include "patterns/organisms/people-module/people-module.html" with people=related_staff title=page.staff_title %}
```

Comment:
**Suggestion**

As staff_title can be blank, I'd default this to "Staff" if it's blank; unless there is a requirement to have it blank even when staff are listed

Reply:
In the model, I set the default value of `staff_title` to "Staff":

https://github.com/torchbox/rca-wagtail-2019/blob/b45074da18b8f03a3abc74e44f0982cfc7c7df1c/rca/programmes/models.py#L372-L377